pages/Identity/Overview.md

```diff
@@ -7,22 +7,319 @@ menu: Identity
+     * @param s ECDSA Signature s value
+     * @param v EIP-155 calculated Signature v value
+     * @param newDelegate Address to delegate new permissions to
+     * @param permission Permission level
+     * 
+     * MUST be signed by owner or other delegate with permissions (implementation specific)
+     * MUST consider newDelegate to be valid from the beginning to time
+     * MUST emit DSNPAddDelegate
+     */
+    function delegateBySignature(bytes32 r, bytes32 s, uint32 v, address newDelegate, Permission permission) external;
+
+    /**
+     * @dev Remove Delegate
+     * @param delegate Address to remove all permissions from
+     * @param endBlock Block number to consider the permissions terminated (MUST be > 0x0). 
+     * 
+     * MUST be called by the delegate, owner, or other delegate with permissions
+     * MUST store endBlock for response in isAuthorizedToAnnounce
+     * MUST emit DSNPRemoveDelegate
+     */
+    function delegateRemove(address delegate, uint64 endBlock) external;
+
+    /**
+     * @dev Remove Delegate By EIP-712 Signature
+     * @param delegate Address to remove all permissions from
+     * @param endBlock Block number to consider the permissions terminated (MUST be > 0x0).
+     * @param r ECDSA Signature r value
+     * @param s ECDSA Signature s value
+     * @param v EIP-155 calculated Signature v value
+     * 
+     * MUST be signed by the delegate, owner, or other delegate with permissions
+     * MUST store endBlock for response in isAuthorizedToAnnounce
+     * MUST emit DSNPRemoveDelegate
+     */
+    function delegateRemoveBySignature(bytes32 r, bytes32 s, uint32 v, address delegate, uint64 endBlock) external;
+
+    /**
+     * @dev Checks to see if address is authorized to announce messages
+     * @param addr Address that is used to test against erecover
+     * @param permission Level of permission check. See Permission for details
+     * @param blockNumber Check for authorization at a particular block number
+     * @return boolean 
+     * 
+     * @dev Return MAY change as deauthorization can revoke past messages
+     */
+    function isAuthorizedToAnnounce(address addr, Permission permission, uint64 blockNumber) external view returns (bool);
+}
+```
+
+
+## Identification
+
+In order to maintain chain level flexibility (see [Networks](/Networks) for more details),
+an identifier outside of the contract address is needed to maintain the graph data integrity without need for a translation.
+The `IIdentification` interface provides the mapping back to the identifier from the contract (see [Registry](/Identity/Registry) for more details) . 
+
+```solidity
+/**
+ * @dev Easy way to map back to the identifier, but must be validated
+ */
+interface IIdentification {
+    /**
+     * @dev Returns the identifier for the contract.
+     * @dev May be validated by testing against (TBD)
+     * 
+     * @return string 
+     */
+    function getId() external view returns (string);
+}
+```
```

Comment:
In a meeting it was discussed that the Registry should allow for reverse resolving, thus removing the need for this interface.